source-hubspot/source_hubspot/streams.py

```diff
@@ -371,7 +372,7 @@ def get_json_schema(self) -> Mapping[str, Any]:
         return json_schema
 
 
-    @retry_401(max_tries=20)
+    @retry_401(max_tries=5)
```

Comment:
Should we keep this at 20, or do we also have a reason for reducing this?

Reply:
The reason for reducing it is to change it back to the original value
This value should not matter anymore, as this backoff should not be triggered again anytime.

Reducing it back to 5 was my way of making sure future reviewers wouldnt think this is being used ( since it would show on their diff this value was modified sometime in the past)